commit e3bc84a5ff775ad26b423f09b6077f8fdca7c644
Author: Daniel Kolesa <daniel@octaforge.org>
Date:   Sun Feb 27 05:29:13 2022 +0100

    default to ld.bfd on powerpc

    (lld does not support --save-restore-funcs)

diff --git a/Makefile b/Makefile
index 74eafb2..5bb8d45 100644
--- a/Makefile
+++ b/Makefile
@@ -444,7 +444,11 @@ KBUILD_HOSTLDLIBS   := $(HOST_LFS_LIBS) $(HOSTLDLIBS)
 CPP		= $(CC) -E
 ifneq ($(LLVM),)
 CC		= clang
+ifeq ($(ARCH),powerpc)
+LD		= ld.bfd
+else
 LD		= ld.lld
+endif
 AR		= llvm-ar
 NM		= llvm-nm
 OBJCOPY		= llvm-objcopy
